triplea_maps.yaml

```diff
@@ -88,10 +88,10 @@
   version: 1
   img: http://tripleamaps.sourceforge.net/images/TripleA_total_world_war_mini.png
   description: |
-    <br><em><b><font size="6">Created by Rolf Larsson and Hepster</font></b></em>
+    <br><em><b style="font-size: 6px;">Created by Rolf Larsson and Hepster</b></em>
```

Comment:
:smile: I'd recommend changing your `game_engine.properties` to point to your locally-modified `triplea_maps.yaml` so you can see how the Download Maps window renders these map descriptions.  That way you can find the new CSS `font-size` (in whatever unit) that is equivalent to the old HTML `font` `size` for the Swing HTML engine.